Traveler.xcodeproj/xcshareddata/xcschemes/Traveler local.xcscheme

```diff
@@ -10,7 +10,7 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Injector Scirpt"
-               scriptText = "#Read through variables file and store variables in associated array (dictionary)&#10;&#10;&#10;FILE=/Users/atanamvari/variables/variables.txt&#10;cat $FILE&#10;declare -gA variables&#10;&#10;if test -f &quot;$FILE&quot;; then&#10;&#10;&#10;while read line&#10;do &#10;key=$(echo $line | cut -d &apos; &apos; -f1)&#10;value=$(echo $line | cut -d &apos; &apos; -f2)&#10;variables[$key]=$value&#10;&#10;done &lt; $FILE&#10;&#10;&#10;#Force git to ignore environment swift file. &#10;git update-index --skip-worktree &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;&#10;#Open environment swift file and search for environment variables&#10;&#10;&#10;cat &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos; | while read line&#10;do&#10;if [[ $line =~ \&quot;\$\(.*\)\&quot; ]]; then&#10;variable=$(echo $line | sed &apos;s/.*&quot;$(\([^&quot;]*\))&quot;.*/\1/&apos;)&#10;echo $variable&#10;echo ${variables[$variable]}&#10;sed -i &apos;&apos; -e &quot;s/\$($variable)/${variables[$variable]}/&quot; &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;fi&#10;done &#10;&#10;cat &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;else &#10;echo &quot;****--Warning: Variables file not found--****&quot;&#10;fi&#10;&#10;"
+               scriptText = "#Read through variables file and store variables in associated array (dictionary)&#10;&#10;&#10;FILE=~/variables/variables.txt&#10;cat $FILE&#10;declare -gA variables&#10;&#10;if test -f &quot;$FILE&quot;; then&#10;&#10;&#10;while read line&#10;do &#10;key=$(echo $line | cut -d &apos; &apos; -f1)&#10;value=$(echo $line | cut -d &apos; &apos; -f2)&#10;variables[$key]=$value&#10;&#10;done &lt; $FILE&#10;&#10;&#10;#Force git to ignore environment swift file. &#10;git update-index --skip-worktree &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;&#10;#Open environment swift file and search for environment variables&#10;&#10;&#10;cat &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos; | while read line&#10;do&#10;if [[ $line =~ \&quot;\$\(.*\)\&quot; ]]; then&#10;variable=$(echo $line | sed &apos;s/.*&quot;$(\([^&quot;]*\))&quot;.*/\1/&apos;)&#10;echo $variable&#10;echo ${variables[$variable]}&#10;sed -i &apos;&apos; -e &quot;s/\$($variable)/${variables[$variable]}/&quot; &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;fi&#10;done &#10;&#10;cat &quot;${SRCROOT}&quot;&apos;/Traveler/Environment/Environment.swift&apos;&#10;else &#10;echo &quot;****--Warning: Variables file not found--****&quot;&#10;fi&#10;&#10;"
```

Comment:
Please update the readme section of this repo so that the variables folder is saved in the right path.